incubator/jenkins-operator/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 0.1.12
+appVersion: 0.2.16
 description: A Helm chart for Kubernetes Jenkins Operator
 name: jenkins-operator
-version: 0.1.12
+version: 0.2.16
```

Comment:
That chart `version` should be different from the `appVersion`. The application and the configuration that can deploy it can change at different rates. Can the `version` be `0.2.0`?